api/managementpb/ia/pagination.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package ia.v1beta1;
+
+option go_package = "api/managementpb/ia;iav1beta1";
+
+// PaginationRequest represents request values for page by page pagination.
+message PaginationRequest {
```

Comment:
Typically `*Request/Response` are top-level entities, used as parameter/response for gRPC methods. Probably we should think about naming.

Reply:
naming reconsidered